update host maps page (#3592)
DataDog/documentation: content/graphing/infrastructure/hostmap.md

  tag: "Graphing"
  text: Understand what is going on at any level of your system
---

## Overview

Host Maps visualize hosts together on one screen, with metrics made comprehensible via color and shape.    

## Usage

### Filter by

`Filter by` limits the Host Map to a specific subset of an infrastructure. The filter input bar in the top left enables filtering of the Host Map by tags as well as Datadog-provided attributes.

> If the filter input bar is empty, the map displays all hosts that are reporting metrics to Datadog.

Reviewer: ```suggestion
If the filter input bar is empty, the map displays all hosts that are reporting the selected metric to Datadog.
```

Example: if you tag your hosts by the environment they are in, you can filter by 'production' to remove hosts in your staging and other environments from the map. If you want to eliminate all but one host role in production, then add that role to the filter, too—the filters are `AND`ed together.

Filterable host attributes (automatically provided):

* `up`: the host is reporting a heartbeat
* `down`: the host is not reporting a heartbeat
* `muted`: Datadog alerts are muted for this host
* `agent`: the host is running the Datadog Agent
* `agent_issue`: often indicates an integration problem, such failed access to a resource

> * `upgrade_required`: the Datadog Agent requires an upgrade

Reviewer: Checked with DataViz, this list is not available in the host map. Lines 30-37 should be deleted.

**Note**: there is a distinction between filtering for `tag:value` and `"tag:value"`—filtering for `tag:value` strictly matches the tag, while filtering for `"tag:value"` performs a search on that text.

### Group hosts by tags

`Group hosts by tags` spatially arranges your hosts into clusters. Any host in a group shares the tag or tags you group by.  

A simple example is grouping your hosts by AWS availability zone. If you add a second grouping tag, such as instance type, then the hosts are further subdivided into groups, first by availability zone and then by instance type, as seen below.

> {{< img src="graphing/infrastructure/hostmap/hostmappart2image2.png" alt="Datadog Host Maps AZ Instance Groups" responsive="true" >}}

Author: the more you know!

{{< img src="graphing/infrastructure/hostmap/blog-host-maps-01.png" alt="Datadog Host Maps Zoom In" responsive="true" style="width:75%;" >}}

### Shapes and colors

By default the color of each host is set to represent the percentage of CPU usage on that host, where the color ranges from green (0% utilized) to red (100% utilized). You can select different metrics from the `Color by` selector.  

Host Maps can also communicate an additional, optional metric with the size of the hexagon: use the `Size by` selector. 

In the screenshot below, the size of the hexagons is the 15 minute average load, normalized so that machines' workloads can be compared even if they have different numbers of cores.

{{< img src="graphing/infrastructure/hostmap/hostmappart2image4.png" alt="Datadog Host Maps Using Color And Size" responsive="true" style="width:80%;">}}

**Note**: The "% CPU utilized" metric uses the most reliable and up-to-date measurement of CPU utilization, whether it is being reported by the Datadog Agent, or directly by AWS or vSphere.

### Display hosts on the Host Map that don't have an Agent installed

> By default, the Host Map only shows hosts that are reporting certain metrics, which can then be used to set a color or size for the individual hexagon within the grid.

Reviewer: ```suggestion
By default, the Host Map only shows hosts that are reporting the selected metric, which can then be used to set a color or size for the individual hexagon within the grid.
```

> Some hosts, like those coming in from Google App Engine, do not pull in these metrics without an Agent running. These hosts can still appear within the Host Map by selecting the "gear" icon on the top-right of the map and enabling "Show hosts with no metrics" in the Host Map settings:

Reviewer: ```suggestion
If a host is not reporting the selected metric, it can still appear within the Host Map by selecting the "gear" icon on the top-right of the map and enabling "Show hosts with no metrics" in the Host Map settings:
```
